plaid/date.go

```diff
@@ -0,0 +1,11 @@
+package plaid
+
+import "time"
+
+// DateLayout is the time format expected by the api
+const DateLayout = "2012-11-01"
```

Comment:
`const DateLayout = "2006-01-02"` 
Date needs to be in go format https://golang.org/pkg/time/#Time.Format

Reply:
Wow, not sure how I messed that up.